browser/ui/BUILD.gn

```diff
@@ -751,6 +751,7 @@ source_set("ui") {
     "//brave/common",
     "//brave/components/ai_chat/core/common/buildflags",
     "//brave/components/ai_rewriter/common/buildflags",
+    "//brave/components/brave_adaptive_captcha",
```

Comment:
So this is needed on Android as well? Suprised it hasn't been an issue before...

Reply:
Only `rewards_panel_ui.cc` used it previously, which is only built on Android. I could have restructured things to keep the dependency Android-only, but it seemed simpler just to remove the condition. What do you think?

Reply:
That makes sense.